packages/dd-trace/src/appsec/standalone.js

```diff
@@ -0,0 +1,41 @@
+'use strict'
+
+const { channel } = require('dc-polyfill')
+const startCh = channel('dd-trace:span:start')
+const { APM_TRACING_ENABLED_KEY, APPSEC_PROPAGATION_KEY } = require('../constants')
+
+let enabled
+
+function onSpanStart ({ span, fields }) {
+  const tags = span.context()?._tags
```

Comment:
btw i'm more scared that "context" is not a function -> `span.context?.()`
same thing in sample() down below

Reply:
it should, span is an instanceof `DatadogSpan`

Reply:
protected